doc/release/1.7.0-notes.rst

```diff
@@ -91,13 +328,637 @@ Backwards incompatible changes
 Other changes
 =============
 
+We now accept and leverage performance improvements from the ahead-of-time
+Python-to-C++ transpiler, Pythran, which can be optionally disabled but is
+enabled by default at build time.
+
+There are two changes to the default behavior of `scipy.stats.mannwhitenyu`:
```

Comment:
This is yet another `mannwhitneyu` paragraph - can we merge then or reference this from the "improvement" part?

Reply:
This has been removed from the paragraph above.